renderer/components/dashboard-projects/menu-list.tsx

```diff
@@ -1,14 +1,34 @@
+import { useState } from 'react';
 import Menu from '@material-ui/core/Menu';
 import MenuItem from '@material-ui/core/MenuItem';
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemText from '@material-ui/core/ListItemText';
+import Collapse from '@material-ui/core/Collapse';
+import ExpandLess from '@material-ui/icons/ExpandLess';
+import ExpandMore from '@material-ui/icons/ExpandMore';
+import Divider from '@material-ui/core/Divider';
 import { ProjectMenuType } from '../../models/dashboard/ProjectMenuType';
+import { menuStyles } from './menu-list.styles';
+import { ProjectDetails } from '../../modules/projects/redux/stepper/data.model';
 
 export default function MenuList(props: {
   state: ProjectMenuType;
   handleClose: () => void;
-  openProjectInIde: () => void;
+  openProjectInIde: (ide: string) => void;
   openProjectDirectory: () => void;
   deleteProject: () => void;

```

Comment:
Move to interface MenuListProps so you can do `MenuList(props: MenuListProps)`

The interface has to be located in the same file